messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/CommonAttributes.java

```diff
@@ -177,9 +177,6 @@
             .setAllowNull(true)
             // do not allow expression as this may reference another resource
             .setAllowExpression(false)
-            .setAlternatives("socket-binding",
-                    "group-address", "group-port",
-                    "local-bind-address", "local-bind-port")
```

Comment:
@TomasHofman you can also remove `"group-address", "group-port", "local-bind-address", "local-bind-port"`from the list of alternatives. They were valid in the legacy messaging subsystem but they no longer exist in the new messaging-activemq subsystem.